src/eBayEnterprise/RetailOrderManagement/Payload/Risk/RiskAssessmentReply.php

```diff
@@ -0,0 +1,150 @@
+<?php
+/**
+ * Copyright (c) 2013-2014 eBay Enterprise, Inc.
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.md.
+ * It is also available through the world-wide-web at this URL:
+ * http://opensource.org/licenses/osl-3.0.php
+ *
+ * @copyright   Copyright (c) 2013-2014 eBay Enterprise, Inc. (http://www.ebayenterprise.com/)
+ * @license     http://opensource.org/licenses/osl-3.0.php  Open Software License (OSL 3.0)
+ */
+namespace eBayEnterprise\RetailOrderManagement\Payload\Risk;
+use eBayEnterprise\RetailOrderManagement\Payload\ISchemaValidator;
+use eBayEnterprise\RetailOrderManagement\Payload\IValidatorIterator;
+use eBayEnterprise\RetailOrderManagement\Payload\IPayloadMap;
+use eBayEnterprise\RetailOrderManagement\Payload\IPayload;
+use eBayEnterprise\RetailOrderManagement\Payload\TTopLevelPayload;
+
+use eBayEnterprise\RetailOrderManagement\Payload\OrderEvents\TOrderEvent;
+
+use Psr\Log\LoggerInterface;
+use Psr\Log\NullLogger;
+class RiskAssessmentReply implements IRiskAssessmentReply
+{
+    use TTopLevelPayload, TOrderEvent;
+    protected $_mockOrderEvent;
+    protected $_responseCode;
+    protected $_sessionId;
+
+    /**
+     * @param IValidatorIterator
+     * @param ISchemaValidator
+     * @param IPayloadMap
+     * @param LoggerInterface
+     * @param IPayload
+     * @SuppressWarnings(PHPMD.UnusedFormalParameter)
+     */
+    public function __construct(
+        IValidatorIterator $validators,
+        ISchemaValidator $schemaValidator,
+        IPayloadMap $payloadMap,
+        LoggerInterface $logger,
+        IPayload $parentPayload = null
+    ) {
+        $this->logger = $logger;
+        $this->validators = $validators;
+        $this->schemaValidator = $schemaValidator;
+        $this->parentPayload = $parentPayload;
+        $this->extractionPaths = [
+            'orderId' => 'string(x:OrderId)',
```

Comment:
i believe this extraction path should be `x:PaymentContext/x:OrderId` also has an extra space in the line indent